lua/plugins.lua

```diff
@@ -59,10 +59,7 @@ return require("packer").startup(function(use)
   -- Autopairs
   use {
     "windwp/nvim-autopairs",
```

Comment:
I vote for adding a `disable` check and to ~~lazyload it on "InsertEnter *" similar to `compe`~~ or maybe that's not the case anymore (?).

Reply:
We used to lazyload both compe and autopairs until users reported problems so we switched to requiring them.  Honestly I think it's possible to lazy load both but that's out of scope for this change.  If that change causes problems, I want to be able to revert a single commit.

Reply:
I'm just going to add an override mechanism.  The disable check should have a different pr so the decision can be made separately.